### PR TITLE
Added documentation for argument text objects #6764

### PR DESCRIPTION
**Pull request by w-cantin**
Hello,

This PR improves the documentation for the ia and aa text objects. It solves the issue in this link #6764
The documentation was lacking and I did not even know those features existed.

Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -660,6 +660,26 @@ Usage examples:
...
+Usage examples:
+
+- `cia` - change the argument under the cursor while preserving separators like comma `,`.
+- `daa` - will delete the whole argument under the cursor and the separators if applicable.
+
+| Setting                             | Description                        | Type        | Default Value |
+| ----------------------------------- | ---------------------------------- | ----------- | ------------- |
+| vim.argumentObjectOpeningDelimiters | A list of opening delimiters       | String list | ["(", "["]    |
+| vim.argumentObjectClosingDelimiters | A list of closing delimiters       | String list | [")", "]"]    |
+| vim.argumen                         | Enable/disable ReplaceWithRegister | String List | [","]         |
```
[the remark this answers is not in the document]
Comment (J-Fields) [reply README.md]: The `Description` is still wrong 🙂

[the remark this answers is not in the document]
Comment (w-cantin) [reply README.md]: Oops I guess I used the wrong file. It should be fixed.

[the remark this answers is not in the document]
Comment (w-cantin) [reply README.md]: Still getting used to git sorry :(. Now it looks ok...

**Comment by w-cantin** [comment]
I made a huge mess with what should have been a simple PR. I am closing this one and gonna reopen one with a much cleaner history.

**Pull request closed without merging**